### PR TITLE
removed copy-webpack-plugin

### PR DESCRIPTION
**Pull request by ramyaparimi**
### Why:

Closes https://github.com/github/docs-engineering/issues/751

### What's being changed:

There are some unnecessary complexity in the frontend build pipeline that involves copying Inter font files from node_modules/primer into the repo. Since those files rarely ever change, it would be nice to simplify the build by removing  this step.

This is done by uninstalling `copy-webpack-plugin` and removing its usage.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,7 +7,7 @@ npm-debug.log
 coverage/
 .linkinator
 /assets/images/early-access
-/assets/fonts/inter
+# /assets/fonts/inter
```
Comment (heiskr) [inline .gitignore:10]: I would delete instead of commenting out, across the pull request. We always have the old code in version history if needed

Comment (ramyaparimi) [reply .gitignore]: Thank you! Deleted the stuff!

**Review by heiskr — APPROVED** [review]

**Comment by Brojai** [comment]
Pada Kha, 5 Ogo 2021 3:36 PG Ramya Parimi ***@***.***>
menulis:

**Comment by ramyaparimi** [comment]
@heiskr the checks keep failing because of the file modified. I dismissed the review but it still fails. Should I be opening this internally?

**Comment by chiedo** [comment]
@ramyaparimi correct! Internally!

**Comment by gracepark** [comment]
Closing as this was merged.

**Pull request closed without merging**